bundles/binding/org.openhab.binding.zwave/database/ge/SmartSwitch.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Minor: I noticed that the database XML file names all seem to be lowercase, like smartswitch.xml.  SmartSwitch.xml might be the only CamelCase file name in the database?